components/faucet-claim.tsx

```diff
@@ -49,7 +49,7 @@ export const FaucetClaim = ({
   const { data: isDowntimeCheck } = useIsDowntime();
 
   const [captchaCode, setCaptchaCode] = useState<string | null>(null);
-  const [showCloudfare, setShowCloudfare] = useState(true);
+  const [setShowCloudflare, setsetShowCloudflare] = useState(true);
```

Comment:
`setsetShowCloudflare` doesn't make sense
convention for `useState` is `x`, `setX`

so this line should be updated to `showCloudflare, setShowCloudflare`